spec/models/chargeback_rate_detail_spec.rb

```diff
@@ -9,6 +9,16 @@
     end
   end
 
+  it "#find_rate" do
```

Comment:
@tledesma 
in it should what this test is going to expect, something like this:
it "finds proper rate according the value"